### ISSUE TITLE
NTR: 5-hydroxymethyluracil high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8809]
New term request 5-hydroxymethyluracil high in urine (segal_030514193847):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000114
- label: Elevated urinary 5-hydroxymethyluracil level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: 
- def: The amount of 5-hydroxymethyluracil in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8809
- synonyms: 5-hydroxymethyluracil high in urine(NONE)
